### PR TITLE
Landscape

### PR DESCRIPTION
**Pull request by borisyankov**
* makes the store 'orientation' aware
* general improvements on padding/margins and touch sizes
* makes padding and sizes of touch areas more consistent
* improves PasswordAuthScreen in Landscape

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@borisyankov, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by neerajwahi** [comment]
Just tried it out on my phone.

When I switch from portrait to landscape back to portrait, section headers are mis-formatted:
screenshot 2017-02-12 14 32 39

Someone else also reported this issue.

**Comment by borisyankov** [comment]
I also reproduced this. Not related to this PR.
Not sure if not even an issue with React Native or the way we style it.

**Comment by neerajwahi** [comment]
Yeah this PR didn't introduce the issue but that's what I originally meant by 'landscape mode is broken'. I'd rather lock the orientation for now if we can't get to the bottom of it.

File: src/message/headers/StreamMessageHeader.js [hunk 1]
```diff
@@ -17,7 +17,6 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     flexDirection: 'row',
     backgroundColor: '#eee',
-    overflow: 'hidden',
```
Comment (neerajwahi) [inline src/message/headers/StreamMessageHeader.js]: We can't do this or the scroll view performance will degrade significantly. Ever child view of scrollview must have 'overflow: hidden' set for the removeClippedSubviews optimization

Comment (neerajwahi) [inline src/message/headers/StreamMessageHeader.js]: Removing 'overflow: hidden' doesn't seem to solve the problem for me anyway. Does it for you?

Comment (borisyankov) [inline src/message/headers/StreamMessageHeader.js]: Ughhh. It seems not. Though in 10 times tested it did?!?
That is better though, then the issue is something else.

File: src/start/PasswordAuthScreen.js [hunk 2]
```diff
@@ -59,11 +59,15 @@ class PasswordAuthScreen extends React.Component {
 
   render() {
     const { email, password, progress, error } = this.state;
+    const { height } = Dimensions.get('window');
```
Comment (neerajwahi) [inline src/start/PasswordAuthScreen.js]: What does this do?

**Pull request closed without merging**